dbfit-java/informix/src/main/java/dbfit/environment/InformixEnvironment.java

```diff
@@ -0,0 +1,416 @@
+package dbfit.environment;
+
+import dbfit.annotations.DatabaseEnvironment;
+import dbfit.api.AbstractDbEnvironment;
+import dbfit.util.DbParameterAccessor;
+import dbfit.util.Direction;
+import dbfit.util.NameNormaliser;
+import dbfit.util.TypeNormaliserFactory;
+
+import java.math.BigDecimal;
+import java.sql.*;
+import java.util.*;
+import java.util.regex.Pattern;
+
+import static dbfit.util.Direction.*;
+
+@DatabaseEnvironment(name="Informix", driver="com.informix.jdbc.IfxDriver")
+public class InformixEnvironment extends AbstractDbEnvironment  {
+
+   @Override
+     public void afterConnectionEstablished() throws SQLException {
+          if(currentConnection.getTransactionIsolation()==0){
+        	  System.out.println("afterConnectionEstablished method");
+        	currentConnection.setAutoCommit(true);
```

Comment:
Yes. I've created the current test DB in `ANSI` mode (it supports transactions) and we need to test in Informix mode too. I was thinking of creating a second test DB for that. One more TODO then.